src/services/http.ts

```diff
@@ -47,34 +46,17 @@ const makeHeaders = (headerOptions) => {
   }
 };
 
-/**
- * Create a request object with the correct URL and headers for the API
- *
- * @param {string} url relative URL from base, ex: `activity/023487` (no leading slash)
- * @param {string} method HTTP method
- * @param {BodyInit} body data to send with request
- * @param {any} headerOptions options to put in the header, if empty only auth is added
- * @returns {Request} A request object
- */
-const createRequest = (url, method, body, headerOptions) =>
-  new Request(`${base}api/${url}`, {
-    method,
-    body,
-    headers: makeHeaders(headerOptions),
-  });
-
 /**
  * Parse an HTTP response
  *
- * @param {Response} res HTTP response
- * @returns {Promise.<Any>} Resolves with response body; rejects on non-2xx
- * response code
+ * @param res The response to parse
+ * @returns The parsed response
```

Comment:
Is it clear from broader context, or perhaps not even true, that the promise "rejects on a non-2xx response code"?

Reply:
No, it's not. I'm not sure why I changed that comment, since it's helpful context for this function. I'll restore it.